server/repository/clientRepository.go

```diff
@@ -0,0 +1,41 @@
+package repository
+
+import (
+	"fmt"
+	"server/model"
+
+	"github.com/google/uuid"
+	"github.com/jinzhu/gorm"
+)
+
+var DB *gorm.DB
+
+func FindClientByID(clientId uuid.UUID) (model.Client, error) {
+	client := model.Client{}
+	if DB.Where("id = ?", clientId).Find(&client).Error != nil {
+		return client, fmt.Errorf("Cannot find client by id: %v", DB.Error)
```

Comment:
when returning errors with `fmt.Errorf` it's best to use `%w` as the error placeholder in the format string instead of `%v` so that it will wrap the underlying error